HoloPipelines/jobs/jobs_io.py

```diff
@@ -80,16 +131,28 @@ def get_logger_for_job(job_id: str):
     return logger
 
 
-def get_log_file_path_for_job(job_id: str):
-    return f"{get_directory_path_for_job(job_id)}/job.log"
+def get_log_file_path_for_job(job_id: str) -> Union[str, None]:
+    """
+    Returns the path to a log file. Looks for current jobs first. If nothing found,
+    looks for finished jobs. If still not found, returns None.
+    """
+    path = get_directory_path_for_job(job_id)
+    if os.path.isdir(path):
+        return f"{path}/job.log"
+    elif os.path.isdir(path.replace(jobs_root, finished_jobs_root)):
+        return f"{path.replace(jobs_root, finished_jobs_root)}/job.log"
+
```

Comment:
An explicit `return None` will be idiomatic given that the function returns different values and None is one of them